credentials/java/com/google/auth/Credentials.java

```diff
@@ -26,6 +26,8 @@
    * populated in headers or other context. The operation can block and fail to complete and may do
    * things such as refreshing access tokens.
    *
+   * <p>If a key ends with {@code "-bin"}, its values are base64-encoded binary data.
+   *
    * @throws IOException if there was an error getting up-to-date access.
```

Comment:
It is not clear how this comment relates to the inputs and outputs of this abstract method. Can you clarify which specific  values have this convention and what that means. It may be that this convention should be described somewhere other than the getRequestMetadata method.